packages/react-components/react-drawer/stories/src/Drawer/index.stories.tsx

```diff
@@ -41,6 +41,7 @@ export default {
     DrawerHeaderTitle,
     DrawerHeaderNavigation,
     DrawerBody,
+    DrawerFooter,
```

Comment:
Even though this is kinda unrelated to the PR itself, `DrawerFooter` was missing here. So I included this in the same PR.